QuizBowlDiscordScoreTracker/Commands/AdminCommandHandler.cs

```diff
@@ -89,11 +89,12 @@ public async Task CheckPermissionsAsync()
 
             if (builder.Length == 0)
             {
-                await this.Context.Channel.SendMessageAsync("All permissions are set up correctly.");
-                return;
+                builder.AppendLine("All permissions are set up correctly.");
```

Comment:
Sending a message with no errors was delayed to after checking if the bot can write to the context channel, as it now possible for the bot to be unable to write messages in the context where it is called, but have all permissions for another channel.

Reply:
Good catch